diff --git a/src/bun.js.zig b/src/bun.js.zig
index 84ab501..934035d 100644
--- a/src/bun.js.zig
+++ b/src/bun.js.zig
@@ -2,6 +2,11 @@ pub const jsc = @import("./bun.js/jsc.zig");
 pub const webcore = @import("./bun.js/webcore.zig");
 pub const api = @import("./bun.js/api.zig");
 pub const bindgen = @import("./bun.js/bindgen.zig");
+pub const embed = @import("./bun.js/embed.zig");
+
+comptime {
+    _ = embed;
+}
 
 pub const Run = struct {
     ctx: Command.Context,
