diff --git a/cmake/targets/BuildBun.cmake b/cmake/targets/BuildBun.cmake
index bf661a6..c505d19 100644
--- a/cmake/targets/BuildBun.cmake
+++ b/cmake/targets/BuildBun.cmake
@@ -827,7 +827,8 @@ if(BUN_LINK_ONLY)
   set_target_properties(${bun} PROPERTIES LINKER_LANGUAGE CXX)
   target_link_libraries(${bun} PRIVATE ${BUN_CPP_OUTPUT})
 elseif(BUN_CPP_ONLY)
-  add_library(${bun} STATIC ${BUN_CPP_SOURCES})
+  add_library(${bun} STATIC ${BUN_CPP_SOURCES} ${BUN_ZIG_OUTPUT})
+  add_dependencies(${bun} bun-zig)
   register_command(
     TARGET
       ${bun}
